app/javascript/src/components/Dashboard/Notes/ListNoteCards.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import Card from "components/Dashboard/Notes/Card";
+
+import { calculateAge, formatTime } from "./utils/prettyPrintTime";
+
+const ListNoteCards = ({ notes = [] }) => {
+  const getCreatedAtText = timeStamp => `Created ${calculateAge(timeStamp)}`;
+  const getDropDownItems = () => [
+    {
+      name: "Edit",
+      func: () => {},
+    },
+    {
+      name: "Delete",
+      func: () => {},
+    },
+  ];
```

Comment:
You could move these utils into a `utils.js` within the closest scope.

Reply:
Fixed with #18